tmt/utils.py

```diff
@@ -954,6 +954,17 @@ def workdir(self) -> Optional[str]:
 
         return self._workdir
 
+    @property
+    def clone_dirpath(self) -> str:
```

Comment:
This is supposed to be a place for keeping various temporary files which we do not want to store long term. If it's not shared or to be used by other `tmt` code, let's use `/tmp` directory instead.